src/dataexplorer/dataexplorer-application/src/components/Schemes/Schemes.style.scss

```diff
@@ -0,0 +1,65 @@
+.site-collapse-custom-panel {
+    background: white;
+    font-size: 24px;
+    border-radius: 0px;
+    margin-bottom: 24px;
+    border: 0px;
+    overflow: inherit;
+    padding-left: 0px;
+    padding-right: 0px;
+    padding-top: 50px;
+}
+
+.site-collapse-custom-collapse {
+    border-radius: 0px;
+    font-size: 10px;
+    background: white;
+    border: 0px;
+    overflow: inherit;
+    margin-bottom: 24px;
+    padding-left: 100px;
+    padding-right: 0px;
```

Comment:
I would use rem instead of px on everything expect for font-size. Rem makes it scalable for different screen sizes